docs/how-to/builtins.rst

```diff
@@ -1,104 +1,138 @@
 Implementing Python Built-ins in JavaScript
 ===========================================
 
+Python's builtins give Python its trademark magical feel. If you're new to Python, read up on them here: https://docs.python.org/3/library/functions.html
+
+Most builtins have already been added to the project, but many do not follow the spec outlined below.
```

Comment:
Avoid colloquial abbreviations; specification, not spev. Although in this case, "description" is probably more accurate; it's not really a formal spec.